<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="3" column="0">
      <widget class="QGroupBox" name="group_box_plot_">
       <property name="title">
        <string>Plot</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>cmap:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_box_cmaps_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>gpGrayscale</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpHot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpCold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpNight</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpCandy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpGeography</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpIon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpThermal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpPolar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpSpectrum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpJet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpHues</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_box_uov_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>U</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>V</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonSaveData">
            <property name="text">
             <string>Save data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveImage">
            <property name="text">
             <string>Save image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="group_box_params_">
       <property name="title">
        <string>Params</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0" colspan="4">
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item row="2" column="1">
           <widget class="QLineEdit" name="le_dv">
            <property name="text">
             <string>0.08</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>F:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_du">
            <property name="text">
             <string>0.16</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="le_k">
            <property name="text">
             <string>0.062</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="le_f">
            <property name="text">
             <string>0.055</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>K:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Du:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Dv:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>custom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-alpha-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-alpha-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-beta-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-beta-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-gamma-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-gamma-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-delta-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-delta-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-epsilon-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-epsilon-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-zeta-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-zeta-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-theta-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-theta-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-eta-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-iota-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-kappa-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-kappa-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-lambda-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-lambda-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-mu-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-mu-02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-xi-01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type-pi-01</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="le_res">
          <property name="text">
           <string>256</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>RES: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="push_button_init_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Init</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>DT:  </string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="line_dt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxPattern">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QComboBox" name="comboBoxMethod">
          <item>
           <property name="text">
            <string>EULER</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SYM-RK2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>141</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="group_box_solver_">
       <property name="title">
        <string>Solver</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_button_run_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBoxParallel">
          <property name="text">
           <string>Parallel</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QSpinBox" name="spin_box_iters_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string extracomment=" iter"> iter</string>
          </property>
          <property name="prefix">
           <string> update: </string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLabel" name="labelCurrTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
